$OpenBSD: patch-configure,v 1.2 2020/07/04 08:23:24 ajacoutot Exp $

ld: error: undefined symbol: _Unwind_Resume
>>> referenced by glib-autocleanups.h:0 (/usr/local/include/glib-2.0/glib/glib-autocleanups.h:0)
>>>               virt_admin-virt-admin.o:(cmdSrvList)

Index: configure
--- configure.orig
+++ configure
@@ -15931,7 +15931,6 @@ fi
        wantwarn="$wantwarn -Wno-unused-command-line-argument"
        ;;
     esac
-    wantwarn="$wantwarn -fexceptions"
     wantwarn="$wantwarn -fasynchronous-unwind-tables"
 
     # Need -fipa-pure-const in order to make -Wsuggest-attribute=pure
